### PR TITLE
[SymfonyBundle] feat: Add support for MaxResults on SSMVault

### PR DESCRIPTION
**Pull request by tyx**
To reduce the number of requests sent to SSM

### CODE REVIEW

File: src/Integration/Symfony/Bundle/tests/Unit/DependencyInjection/ConfigurationTest.php [hunk 1]
```diff
@@ -39,6 +39,7 @@ public function testDefaultValues(): void
                     'pool' => 'cache.system',
                     'ttl' => 600,
                 ],
+                'max_results' => null,
```
Comment (jderusse) [inline src/Integration/Symfony/Bundle/tests/Unit/DependencyInjection/ConfigurationTest.php:42]: Are you sure that `null`, won't be sent in the request (and AWS won't reject the payload)?

**Review by jderusse — APPROVED** [review]
Just a minor question, otherwise LGTM

File: src/Integration/Symfony/Bundle/tests/Unit/DependencyInjection/ConfigurationTest.php (hunk 1 above)
Comment (tyx) [reply src/Integration/Symfony/Bundle/tests/Unit/DependencyInjection/ConfigurationTest.php:42]: According to the `GetParametersByPathRequest` it should not be a problem. `MaxResults` is null by default.

https://github.com/async-aws/aws/blob/125a4b95d49fff611f24ce7f6150ba4528727bd8/src/Service/Ssm/src/Input/GetParametersByPathRequest.php#L88

So I guess SSMClient already manage it correctly no ?

**Comment by jderusse** [comment]
Thank you @tyx

**Pull request merged**